<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FFTOptions</class>
 <widget class="QWidget" name="FFTOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>194</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="GroupBoxFFTOptions">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>FFT Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QGridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="3">
         <widget class="QComboBox" name="_apodizeFunction">
          <item>
           <property name="text">
            <string comment="Use the application default option">Default</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Bartlett</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Blackman</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Connes</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>cosine</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gaussian (Custom Sigma)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hamming</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Hann</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Welch</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="_sigmaLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>S&amp;igma:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>_sigma</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QDoubleSpinBox" name="_sigma">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="_apodize">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="whatsThis">
           <string>Select from this pulldown the source vector for the Curve's y axis error bars.  If &quot;&lt;None&gt;&quot; is selected, there will be no y axis error bars drawn.  Create new vectors with the &quot;Vectors&quot; tab.</string>
          </property>
          <property name="text">
           <string>Apodi&amp;ze</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="textLabel2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Fun&amp;ction:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>_apodizeFunction</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="_removeMean">
          <property name="text">
           <string>Remo&amp;ve mean</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="2">
         <widget class="QLabel" name="_FFTLengthLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>FFT &amp;length: 2^</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>_FFTLength</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QSpinBox" name="_FFTLength">
          <property name="whatsThis">
           <string>Sets the length of the FFTs in generating the power spectrum.  2^10 = 1024, etc.  The larger the number, the larger the noise and resolution.</string>
          </property>
          <property name="minimum">
           <number>4</number>
          </property>
          <property name="maximum">
           <number>24</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="_interleavedAverage">
          <property name="text">
           <string>Interleave&amp;d average</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QCheckBox" name="_interpolateOverHoles">
          <property name="text">
           <string>Interpolate over &amp;holes</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="TextLabel1_9_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Sa&amp;mple rate:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>_sampleRate</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_sampleRate">
          <property name="whatsThis">
           <string>The sample rate at which the measurements were acquired, for proper normalization.</string>
          </property>
          <property name="text">
           <string notr="true">60.0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TextLabel1_9_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="whatsThis">
           <string>Apply new data sample specifications to selected vectors.</string>
          </property>
          <property name="text">
           <string>Vector units:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>_vectorUnits</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_vectorUnits">
          <property name="whatsThis">
           <string>The units of the Data Vector - for automatic labels.</string>
          </property>
          <property name="text">
           <string comment="Abbreviation for Volts">V</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="TextLabel1_9_2_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Rate units:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>_rateUnits</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_rateUnits">
          <property name="whatsThis">
           <string>The units for the sample rate.  Used for automatic labels.</string>
          </property>
          <property name="text">
           <string comment="Abbreviation for Hertz (unit)">Hz</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Ou&amp;tput:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="buddy">
           <cstring>_output</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="_output">
          <item>
           <property name="text">
            <string>Amplitude Spectral Density (V/Hz^1/2)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Power Spectral Density (V^2/Hz)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Amplitude Spectrum (V)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Power Spectrum (V^2)</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>_removeMean</tabstop>
  <tabstop>_apodize</tabstop>
  <tabstop>_apodizeFunction</tabstop>
  <tabstop>_sigma</tabstop>
  <tabstop>_interleavedAverage</tabstop>
  <tabstop>_FFTLength</tabstop>
  <tabstop>_interpolateOverHoles</tabstop>
  <tabstop>_sampleRate</tabstop>
  <tabstop>_vectorUnits</tabstop>
  <tabstop>_rateUnits</tabstop>
  <tabstop>_output</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
